UI/Web/src/app/admin/manage-settings/manage-settings.component.html

```diff
@@ -33,6 +33,19 @@
             </div>
         </div>
 
+        <div class="mb-3">
+            <label for="settings-baseurl" class="form-label">Base Url (WARNING: Requires server restart)</label>&nbsp;<i class="fa fa-info-circle" placement="right" [ngbTooltip]="baseUrlTooltip" role="button" tabindex="0"></i>
+            <ng-template #baseUrlTooltip>Use this if you want to host Kavita on a base url ie) yourdomain.com/kavita (WARNING: Requires server restart)</ng-template>
```

Comment:
Remove the  (WARNING: Requires server restart) and instead add it to the top where it mentions Port change, etc needs server restart.